python/GafferSceneUI/SceneInspector.py

```diff
@@ -963,6 +969,15 @@ def __menuDefinition( self, widget = None ) :
 					}
 				)
 
+			if self.__inspector.supportsShaderParameters( target ) :
+
+				m.append(
+					"/Show Shader Parameters",
```

Comment:
I think this is behaving a bit oddly because of the `target/nodes` mixup : it'd be nice for things to work like this when everything is fixed up :
 
- When right clicking on one side of the diff, open the window for just the corresponding target.
- When using the gear icon, have menu items for inspecting just A, just B or comparing them.

Reply:
That's how it should behave now. I didn't know about the context menu on the diff. Thanks for pointing that out.